src/index.js

```diff
@@ -480,6 +483,8 @@ export default class ScrollBooster {
             this.scrollOffset.x = -event.deltaX;
             this.scrollOffset.y = -event.deltaY;
 
+            this.props.onWheel(state, event);
```

Comment:
Maybe move it to start of the function, before condition, so it can be used without `emulateScroll` option?

Reply:
I'm not sure, but in this case my modifications `this.scrollOffset` will be override by default, correct? Callback function must be after:
```
            this.scrollOffset.x = -event.deltaX;
            this.scrollOffset.y = -event.deltaY;
```
or it will be useless.

Reply:
Oh you right, thanks :) Merging it then